main.go

```diff
@@ -131,6 +128,10 @@ func CliMain(cm *CloudManager, ops *Options, dm GameDefManager, channels *Channe
 			result, err := cm.PerformSyncOperation(storage, syncops, syncpath.Path, remotePath)
 			if err != nil {
 				fmt.Println(err)
+				logs <- Message{
+					Err:      err,
+					Finished: true,
+				}
```

Comment:
This is the cause of a number of "hanging" sync screens that users have reported.